app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -1031,8 +1031,20 @@ class PatientSummaryScreenControllerTest {
   fun `when the link id with patient is cancelled, the patient summary screen must be closed`() {
     val openIntention = OpenIntention.LinkIdWithPatient(identifier = Identifier("id", BpPassport))
     uiEvents.onNext(PatientSummaryScreenCreated(patientUuid, openIntention, Instant.now(clock)))
+
     uiEvents.onNext(PatientSummaryLinkIdCancelled)
 
     verify(screen).goToPreviousScreen()
   }
+
+  @Test
+  fun `when the link id with patient is completed, the link id screen must be closed`() {
+    val openIntention = OpenIntention.LinkIdWithPatient(identifier = Identifier("id", BpPassport))
+    uiEvents.onNext(PatientSummaryScreenCreated(patientUuid, openIntention, Instant.now(clock)))
+
+    uiEvents.onNext(PatientSummaryLinkIdCompleted)
+
+    verify(screen).hideLinkIdWithPatientSheet()
```

Comment:
Can you also add another verify ->
`verify(screen, never()).showLinkIdWithPatientSheet(any(), any())`

Reply:
That verification will always fail because the sheet must always be shown when the intention is `LinkIdWithPatient`.